plugins/joke.lua

```diff
@@ -13,8 +13,8 @@ return {
 			if not first then
 				first = sender[1]
 			end
-			local h, s = assert(http_request.new_from_uri(
-			        "https://api.icndb.com/jokes/random?limitTo=nerdy&firstName="..first.."&lastName="):go())
+			local h, s = http_request.new_from_uri(
```

Comment:
That assumes the caller wants falsey + error message => which it doesn't.
It's up to a plugin to do it's own error reporting (e.g. might want to send to channel/user who requested; or may just want to print to `stdout` or `stderr`)